testgrid/config/config.yaml

```diff
@@ -1826,6 +1835,17 @@ dashboards:
   - name: kopeio-upup-aws
     test_group_name: kopeio-kubernetes-e2e-upup-aws
 
+- name: multi-platform
+  dashboard_tab:
+  - name: periodic-kubeadm-gce-amd64
```

Comment:
Strongly suggest choosing as short a name as is reasonable (`gce` or `gce-amd64`) so that you can actually see your test results in the dashboard instead of the tab names taking up 50% of the screen real estate

Reply:
can make shorter later when I've discussed and synced on the naming with others, want to see whether I'm on the right track first :)